### ISSUE TITLE
Send all relevant metadata for all content types to Cinder when we report

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#21234]
See https://docs.google.com/document/d/1k1iyC1TMdyaTsOGlJLJpDyXr4l03O9bGemOSHrbKbnE/edit#heading=h.dca2g7q1cvvo for what metadata needs to be sent with each content type.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDSRV-506)

**Comment by wagnerand** [comment]
@eviljeff I chatted with Lindsay about this, she'll include that in her proposal.

**Comment by eviljeff** [comment]
Should we also include the version for add-ons, if available?

**Comment by diox** [comment]
Note: the question about whether or not to send all translations also applies to collections, which potentially have name & description in several locales

**Comment by ioanarusiczki** [comment]
@diox 

I checked on -dev for extensions and themes so far. For both I don't think I've noticed the Author in Cinder. 

Both have also ID, slug and guid in Cinder (the doc does not mention those but I think these should be present)

Themes don't have release notes or theme icon - there is only a preview. (the doc mentions them both as required)

**Comment by ioanarusiczki** [comment]
@diox 

- I found details about reported/reporter in Relashionships

- ratings/developer replies, I see for both type: amo rating

- collections: I did not find collection items, name, link, author, summary or item rating. Docs metion a badge but AMO collections don't have badges

- user profiles: I did not see biography field in Cinder,  I reported id 10641572. Average ratings in Cinder: 3. Average ratings on amo: 3.8. Could not check for other users after it because the reports did not arrive in Cinder , I'll re-check tomorrow.
